### ISSUE TITLE
Clarification on loss function names

### ISSUE THREAD

**Issue by bkkaggle** [google/objax#89]
Hi,

I'm assuming `cross_entropy_logits()` is meant to be used on the raw logits of the model and `sigmoid_cross_entropy_logits()` is the version that's meant to be used on logits that have been normalized to [0, 1] with a sigmoid or softmax layer? 

Or is that `sigmoid_cross_entropy_logits()` is meant to be used for binary classification on the raw logits of the model and `cross_entropy_logits()` is meant for multi-class classification?

**Comment by david-berthelot** [comment]
Your second interpretation is the right one: sigmoid for binary, cross for multiclass.
